e107_admin/plugin.php

```diff
@@ -665,51 +680,59 @@ private function pluginDownload()
 		$frm = e107::getForm();
 		$mes = e107::getMessage();
 		
+		e107::getDebug()->log("Plugin Download.");
+		$tmp = explode('&amp;', e_QUERY); // #1712 we need to process new ampersand separated query
```

Comment:
I can't reproduce any of these issues on my local machine. If you can grant me FTP access to a machine that has this issue, I could run some tests. I have a hunch it is the use of '&amp' in the URL. I will commit that change and we can see how that affects things.